README.md

```diff
@@ -25,67 +22,68 @@ import (
 	"fmt"
 	"time"
 
-	"github.com/jasonlvhit/gocron"
+	"github.com/go-co-op/gocron"
 )
 
 func task() {
-	fmt.Println("I am running task.")
+    fmt.Println("I am running task.")
 }
 
 func taskWithParams(a int, b string) {
-	fmt.Println(a, b)
+    fmt.Println(a, b)
 }
 
 func main() {
-	// Do jobs without params
-	gocron.Every(1).Second().Do(task)
-	gocron.Every(2).Seconds().Do(task)
-	gocron.Every(1).Minute().Do(task)
-	gocron.Every(2).Minutes().Do(task)
-	gocron.Every(1).Hour().Do(task)
-	gocron.Every(2).Hours().Do(task)
-	gocron.Every(1).Day().Do(task)
-	gocron.Every(2).Days().Do(task)
-	gocron.Every(1).Week().Do(task)
-	gocron.Every(2).Weeks().Do(task)
-
-	// Do jobs with params
-	gocron.Every(1).Second().Do(taskWithParams, 1, "hello")
-
-	// Do jobs on specific weekday
-	gocron.Every(1).Monday().Do(task)
-	gocron.Every(1).Thursday().Do(task)
-
-	// Do a job at a specific time - 'hour:min:sec' - seconds optional
-	gocron.Every(1).Day().At("10:30").Do(task)
-	gocron.Every(1).Monday().At("18:30").Do(task)
-	gocron.Every(1).Tuesday().At("18:30:59").Do(task)
-
-	// Begin job immediately upon start
-	gocron.Every(1).Hour().From(gocron.NextTick()).Do(task)
-
-	// Begin job at a specific date/time
-	t := time.Date(2019, time.November, 10, 15, 0, 0, 0, time.Local)
-	gocron.Every(1).Hour().From(&t).Do(task)
-
-	// NextRun gets the next running time
-	_, time := gocron.NextRun()
-	fmt.Println(time)
-
-	// Remove a specific job
-	gocron.Remove(task)
-
-	// Clear all scheduled jobs
-	gocron.Clear()
-
-	// Start all the pending jobs
-	<- gocron.Start()
-
-	// also, you can create a new scheduler
-	// to run two schedulers concurrently
-	s := gocron.NewScheduler()
-	s.Every(3).Seconds().Do(task)
-	<- s.Start()
+    
+    s1 := gocron.NewScheduler(time.UTC)
+    s1.Every(3).Seconds().Do(task)
+    <- s1.Start() // starts running (blocks current thread)
+
+    // Do jobs without params
+    s2 := gocron.NewScheduler(time.UTC)
+    s2.Every(1).Second().Do(task)
+    s2.Every(2).Seconds().Do(task)
+    s2.Every(1).Minute().Do(task)
+    s2.Every(2).Minutes().Do(task)
+    s2.Every(1).Hour().Do(task)
+    s2.Every(2).Hours().Do(task)
+    s2.Every(1).Day().Do(task)
+    s2.Every(2).Days().Do(task)
+    s2.Every(1).Week().Do(task)
+    s2.Every(2).Weeks().Do(task)
+
+    // Do jobs with params
+    s2.Every(1).Second().Do(taskWithParams, 1, "hello")
+
+    // Do jobs on specific weekday
+    s2.Every(1).Monday().Do(task)
+    s2.Every(1).Thursday().Do(task)
+
+    // Do a job at a specific time - 'hour:min:sec' - seconds optional
+    s2.Every(1).Day().At("10:30").Do(task)
+    s2.Every(1).Monday().At("18:30").Do(task)
+    s2.Every(1).Tuesday().At("18:30:59").Do(task)
+
+    // use .From(gocron.NextTick()) to run job immediately upon start
+    s2.Every(1).Hour().From(gocron.NextTick()).Do(task)
+
+    // Begin job at a specific date/time. 
+    // Attention: scheduler timezone has precedence over job's timezone!
```

Comment:
How does it work if scheduler i UTC and they schedule task From 10AM UTC+2? does it do 10AM UTC?

Reply:
10AM UTC... .From() gets confusing because of this

Reply:
It's just confusing that the From func also allows passing a timezone, which will then be overridden by the scheduler. The logic makes sense and I am good with it. Just wondering if we can disallow passing a timezone? Probably not because it takes a time.

Reply:
Awkward right? The only way that I can think of is actually to pass a strict own by the library...